proto/net.proto

```diff
@@ -106,6 +108,53 @@ message Weights {
     optional Layer ln2_betas = 6;
   }
 
+  message PolicyHead {
+    // Policy head
+    // Extra convolution for AZ-style policy head
+    optional ConvBlock policy1 = 100;
+    optional ConvBlock policy = 101;
+    // Add policy encoders for legacy support
+    repeated EncoderLayer pol_encoder = 102;
+    optional uint32 pol_headcount = 103;
+    optional Layer ip_pol_w = 1;
+    optional Layer ip_pol_b = 2;
+    // For policy attention, up to and including NETWORK_SE_WITH_HEADFORMAT the
+    // "embedding" activation is SELU, otherwise it is the default activation.
+    optional Layer ip2_pol_w = 3;  // "wq" in policy attention
+    optional Layer ip2_pol_b = 4;
+    optional Layer ip3_pol_w = 5;  // "wk" in policy attention
+    optional Layer ip3_pol_b = 6;
+    optional Layer ip4_pol_w = 7;  // "ppo" in policy attention
+  }
+
+  message ValueHead {
+    // Value head
+    optional ConvBlock value = 100;
+    optional Layer ip_val_w = 1;  // "embedding" for attention body value
+    optional Layer ip_val_b = 2;
+    optional Layer ip1_val_w = 3;
+    optional Layer ip1_val_b = 4;
+    optional Layer ip2_val_w = 5;
+    optional Layer ip2_val_b = 6;
+    optional Layer ip_val_err_w = 7;
+    optional Layer ip_val_err_b = 8;
+  }
+
+  message PolicyHeads {
+    optional Layer ip_pol_w = 1;    // "embedding" in policy attention
+    optional Layer ip_pol_b = 2;
```

Comment:
Since those two are shared among all policy heads, why not just use the original ones?

Reply:
An alternative is to rename them to avoid confusion (if we keep the corresponding fields in `PolicyHead`).

Reply:
Agree to renaming.

Reply:
These have been kept as is, per the latest updates in LeelaChessZero/lc0#1915